flink-end-to-end-tests/test-scripts/common_docker.sh

```diff
@@ -33,8 +33,25 @@ function containers_health_check() {
   done
 }
 
-function build_image_with_jar() {
-    local job_artifacts=$1
-    local image_name=${2:-flink-job}
-    ./build.sh --from-local-dist --job-artifacts ${job_artifacts} --image-name ${image_name}
+function build_image() {
+    local image_name=${1:-flink-job}
+
+    echo "Starting fileserver for Flink distribution"
+    pushd ${FLINK_DIR}/..
+    tar -czf "${TEST_DATA_DIR}/flink.tgz" flink-*
+    popd
+    pushd ${TEST_DATA_DIR}
+    python -m SimpleHTTPServer 9999 &
```

Comment:
Given that we are primarily interested in the running on CI I'm fine with the current state.
It can blow up in any case since you can't guarantee python (2 or 3) to exist at all; the only safe solution being a pure bash/java version, bash naturally not being realistic option and Java not really being worth the additional effort, and it would also make it way harder to offer a script in that case.
Originally I wanted to serve things via netcat but it didn't quite work.

Dockerizing this one thing seems a bit overkill to me.

Reply:
You don't need to create your own docker image.
```
docker run -it -p 9999:9999 -v `pwd`:/data python:3.7.7-slim-buster python -m http.server 9999
```

Reply:
I'd prefer not having to download another 60mb for this menial task; that alone would imo discard it as a viable option as a utility sript.